README.md

```diff
@@ -112,6 +112,8 @@ Please find the list of all configuration options, including descriptions, below
 | `REMEMBER_USERS_FOR_HOURS` | `--remember-users-for-hours` | No | How long between back-filling attempts for non-followed accounts? Defaults to `168`, i.e. one week.
 | `HTTP_TIMEOUT` | `--http-timeout` | No | The timeout for any HTTP requests to the Mastodon API in seconds. Defaults to `5`.
 | -- | `--lock-hours` | No | Determines after how many hours a lock file should be discarded. Not relevant when running the script as GitHub Action, as concurrency is prevented using a different mechanism. Recommended value: `24`.
+| -- | `--lock-file` | No | Location for the lock file. If not specified, will use `lock.lock` under the state directory. Not relevant when running the script as GitHub Action.
+| -- | `--state-dir` | No | Directory storing persistent files, and the default location for lock file. Changing just this parameter would break the rest of the GitHub Action.
```

Comment:
Sorry to be a pain, but can we just remove this sentence, and replace with the usual 'Not relevant when running the script as GitHub Action.':

> Changing just this parameter would break the rest of the GitHub Action.

It sounds scary, and since we haven't exposed it in the workflow .yml file it's impossible to change in the GitHub Action anyway.

Reply:
Ok, changed it.

Reply:
Thanks @arachnist !

I will merge this for the next release